### PR TITLE
feat: send btc disabled kill switch

### LINKED ISSUE #3130: [Bitcoin] Fix kill switch with dev builds
Wallet config to disable bitcoin was removed bc it wasn't working with dev builds to test.

### LINKED ISSUE #3201: Convert Bitcoin feature flag into remote config
Currently the user can toggle Bitcoin functionality in the console with `debug.setBitcoinFeature(true)`

We should convert this user-facing option into a remote configuration for us to turn off and on for users as a team, in preparation for roll-out.

Note that the "Add now" option for inscriptions under "Collectibles" shows even when Bitcoin is turned off, so it should be brought into the coverage of the feature flag as well so it doesn't appear unless flipped on.

Image

And we should take care not to show the collectibles area in general if Bitcoin is turned off and there are no Stacks NFTs to show there per https://github.com/hirosystems/stacks-wallet-web/issues/3200

### PR DESCRIPTION
**Pull request by fbwoolf**
> Try out this version of the Hiro Wallet - download [extension builds](https://github.com/hirosystems/stacks-wallet-web/actions/runs/4218549361).

This PR adds a kill switch to disable bitcoin sends in our wallet config and routes users to an alert/error page with help links...

Screen Shot 2023-02-18 at 7 16 53 PM

### CODE REVIEW

**Comment by markmhendrickson** [comment]
Looking good! One small improvement here might be to remove the "Still stuck?" part since presumably they're stuck if they see this. We can keep the support link in case they have an acute need to contact us about being stuck. cc @314159265359879

File: src/app/query/common/hiro-config/hiro-config.query.ts [hunk 1]
```diff
@@ -87,10 +87,9 @@ export function useHasFiatProviders() {
   );
 }
 
-// ts-unused-exports:disable-next-line
-export function useConfigBitcoinFeatureEnabled() {
+export function useConfigBitcoinEnabled() {
   const config = useRemoteHiroConfig();
-  return config?.bitcoinFeatureEnabled;
+  return config?.bitcoinEnabled;
```
Comment (kyranjamie) [inline src/app/query/common/hiro-config/hiro-config.query.ts]: Would add a default to true, so that if github is down, we don't block users

**Comment by kyranjamie** [comment]
Build errors, but looks good. Guess we'll hold off merging until last moment?

**Comment by fbwoolf** [comment]
> Build errors, but looks good. Guess we'll hold off merging until last moment?

Yeah, prob best just in case. You agree @markmhx. This will just enable btc on dev mainnet ...but maybe not too risky at this point? If we are going to wait, I might just add conditionally running the new send form tests here based on the config feature flag?

**Comment by fbwoolf** [comment]
Almost forgot to handle Ledger again! Haha

**Comment by fbwoolf** [comment]
I changed the base branch here to be `refactor/kill-old-send-form` so I can merge this into that work, otherwise the integration tests won't pass here.

**Pull request merged**